queries/elixir.scm

```diff
@@ -0,0 +1,41 @@
+(comment) @comment
+(anonymous_function) @anonymousFunction
+(arguments) @arguments_list
+(call
+  target: (_) @functionCallee) @functionCall @functionCallee.domain
+
+;;(keywords
+;;  (pair
+;;    key: (_) @collectionKey @collectionItem.trailing.end.startOf
+;;    value: (_) @value) @collectionItem @collectionItem.trailing.start.endOf
+;;    (",")? @collectionItem.trailing.start.startOf
+;;    (#allow-multiple! @collectionItem))
+
+;; How does surrounding whitespace work for this
+(map_content
+  (binary_operator
+      left: (_) @collectionKey @collectionKey.trailing.start.endOf
+      right: (_) @value @collectionKey.trailing.end.startOf) @collectionItem
+      ","? @collectionItem.trailing
+      (#shrink-to-match! @collectionItem "([^,]+)"))
+
+(call
+  target: (identifier) @ignore
+  (arguments
+    [
+      ; zero-arity functions with no parentheses
+      (identifier) @functionName
+      ; regular function clause
+      (call target: (identifier) @functionName)
+      ; function clause with a guard clause
+      (binary_operator
+        left: (call target: (identifier) @functionName)
+        operator: "when")
+    ])
+    (do_block
+      .
+      "do" @namedFunction.interior.start.endOf
+      .
+      "end" @namedFunction.interior.end.startOf
+      .)?
+  (#match? @ignore "^(def|defp|defdelegate|defguard|defguardp|defmacro|defmacrop|defn|defnp)$"))  @namedFunction @functionName.domain
```

Comment:
Does this predicate do anything? I've never used it before, but I think you're right that it should work out of the box

Reply:
I pulled it from the [`ts-elixir` repo](https://github.com/elixir-lang/tree-sitter-elixir/blob/a2861e88a730287a60c11ea9299c033c7d076e30/queries/tags.scm#L10), it seems to be working

Reply:
nice. Would be worth a negative test using "cursorless record error", where you try saying "change funk" from something that is a function call that uses a function other than eg `def` to ensure it isn't treated as `"funk"`, if that makes sense